slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -142,8 +187,8 @@ ::grpc::Status SLAMServiceImpl::GetJpegMap(const GetMapRequest *request,
         }
     } catch (std::exception &e) {
         std::ostringstream oss;
```

Comment:
[nit] remove unused variable (or log this instead)

Reply:
Can we add compiler warnings to fail the build if we have unused variables? Checking for this seems like something a compiler would be good at.